### PR TITLE
Added basic checks to ensure the correct version of Docker Compose is installed.

### PR DESCRIPTION
**Pull request by felix-caboff**
Safety check to ensure the correct version of docker compose is installed.  Used the existing docker detection logic to ensure that the correct version is found.

### CODE REVIEW

**Comment by edznux-dd** [comment]
Hey thanks for the PR.
I'm not sure the string matching is correct?
I have `Docker Compose version v2.21.0` on my machine, so it doesn't match `grep '^Docker Compose version 2'` what do you have on yours?

File: scripts/kubehound.sh [hunk 1]
```diff
@@ -31,6 +31,12 @@ check_docker() {
         exit 1
     fi
 
+    if ! [ "$(docker compose version | grep '^Docker Compose version 2')" ]; then
```
Comment (edznux-dd) [inline scripts/kubehound.sh:34]:
```suggestion
    if ! [ "$(docker compose version | grep '^Docker Compose version v2')" ]; then
```
I think?

**Review by edznux-dd — CHANGES_REQUESTED** [review]

File: scripts/kubehound.sh [hunk 2]
```diff
@@ -31,6 +31,12 @@ check_docker() {
         exit 1
     fi
 
+    if ! [ "$(docker compose version | grep '^Docker Compose version v2')" ]; then
```
Comment (d0g0x01) [inline scripts/kubehound.sh:34]: what about when v3 is released?

**Comment by edznux-dd** [comment]
Hey, we have reworked a bit the Makefile to be *way* more user friendly as part of #200!

It is available as part of the pre-release already but we will also ship a "real version" soon.

Closing that PR, thanks for the work though!
And sorry for the delay here.

**Pull request closed without merging**